Braintree.xcodeproj/project.pbxproj

```diff
@@ -834,7 +848,6 @@
 		42ED470123A3E9DB00B889A8 /* FakeBundle.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = FakeBundle.swift; sourceTree = "<group>"; };
 		42ED470323A3E9F600B889A8 /* FakeDevice.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = FakeDevice.swift; sourceTree = "<group>"; };
 		42F2FF2E2333D2B20083CA10 /* BTAuthenticationInsight_Tests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = BTAuthenticationInsight_Tests.swift; sourceTree = "<group>"; };
-		42F3F6DA2603B81C00401B0D /* CardinalMobile.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = CardinalMobile.xcframework; path = Frameworks/CardinalMobile.xcframework; sourceTree = "<group>"; };
```

Comment:
Doesn't seem like this should have been deleted?

Reply:
I don't think it should have, nope. Will look into why it was deleted on monday. Very well could have been my mistake as I was adding PayPalCheckout.

Reply:
@jonathajones I'm going to add this line back in, and then move the work from your fork over to the `braintree:pp-native-xo` branch, so we can start making PRs for the tests & demo.

There will be more opportunities for other folks (Susan & David) once they're back in the office to review this PR before it goes into master.

Let me know if that sounds OK and I'll go ahead!